### ISSUE TITLE
License for the repo

### ISSUE THREAD

**Issue by lonvia** [geocoders/geocoder-tester#60]
I just noticed that the repo hasn't yet any licensing information. I think it would better to clear that up.

@yohanboniface do you have opinion what it should be?

**Comment by yohanboniface** [comment]
Not really! For me, the more open, the better. I usually use something like WTFPL, but it's not always well understood ;)

Do you have a preference ?

**Comment by lonvia** [comment]
Something permissive is fine with me. How about something like MIT or BSD, just to have something people usually know, and add the exemption that all test cases are considered public domain.

**Comment by yohanboniface** [comment]
Sounds good to me! :)

And cheers for the new year, Sarah :)
